addons/mail/static/tests/helpers/mock_server.js

```diff
@@ -1523,6 +1526,26 @@ MockServer.include({
             "active": partner.active,
         };
     },
+    /**
+     * Simulates `/mail/get_suggested_recipients` route.
+     *
+     * @private
+     * @returns {Object} args
+     * @returns {Array}
+     */
+    _mockGetSuggestedRecipient(args) {
+        let partners = this._getRecords(
+            args.model,
```

Comment:
Why not hard-coded `res.partner`? Just feels like we have to trust flow and naming of variables, which is quite weak IMO.